internal/rpc/chat/user.go

```diff
@@ -113,6 +119,96 @@ func (o *chatSvr) FindUserPublicInfo(ctx context.Context, req *chat.FindUserPubl
 	}, nil
 }
 
+func (o *chatSvr) AddUserAccount(ctx context.Context, req *chat.AddUserAccountReq) (*chat.AddUserAccountResp, error) {
```

Comment:
It would be more efficient to split this into multiple functions for better handling:
1. Unique identifier for phone numbers
2. Unique identifier for email addresses
3. Unique identifier for UserIDs.

Reply:
ok